app/controllers/performance_dashboards/overview_controller.rb

```diff
@@ -30,6 +30,10 @@ def details
       end
     end
 
+    def download
+      render xlsx: 'xlsx_download', filename: "Client Performance.xlsx"
```

Comment:
Need to check, I think this needs to happen in the household controller as well, or potentially get a dynamic file name.